UnityProjects/MRTKDevTemplate/Assets/Shaders/Text3DShader.shader

```diff
@@ -0,0 +1,180 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// NOTE: MRTK Shaders are versioned via the MRTK.Shaders.sentinel file.
+// When making changes to any shader's source file, the value in the sentinel _must_ be incremented.
+
```

Comment:
I don't think we need this shader.  The `Graphics Tools/Text Mesh Pro` shader should do all the text renderer stuff we want